### PR TITLE
Generating the netCDF to COG statistics for validation

### PR DESCRIPTION
**Pull request by SwordSaintLancelot**
## **Motivation**
Multiple datasets have been ingested in the GHG dashboard. In order to validate the transformed data, we generate statistical report from raw and transformed data.

## **Changes**
Creating the scripts to read the raw and transformed files, generate the JSON for the statistics and create plots for better visual understanding of the raw and transformed data.

### CODE REVIEW

File: generating_statistics_for_validation/tm54dvar-ch4flux-monthgrid-v1/stats_summary.png
Comment (siddharth0248) [inline generating_statistics_for_validation/tm54dvar-ch4flux-monthgrid-v1/stats_summary.png:1]: raw distribution doesn't have values?
can you bring the title for the plots in two rows, they are overlapping

File: generating_statistics_for_validation/oco2geos-co2-daygrid-v10r/stats_summary.png
Comment (siddharth0248) [inline generating_statistics_for_validation/oco2geos-co2-daygrid-v10r/stats_summary.png:1]: COG distribution doesn't have values?
can you bring the title for the plots in two rows, they are overlapping

Comment (siddharth0248) [reply generating_statistics_for_validation/oco2geos-co2-daygrid-v10r/stats_summary.png:1]: Overall distribution doesn't match

Comment (SwordSaintLancelot) [reply generating_statistics_for_validation/oco2geos-co2-daygrid-v10r/stats_summary.png:1]: I saw there were some issues with the plots generated. I am working on them and will update them soon.

**Review by siddharth0248 — APPROVED** [review]

**Pull request merged**